app/views/styles/component/_form_footer.scss

```diff
@@ -147,12 +147,6 @@
             margin-top: 0;
         }
     }
-
-    &.end {
-        .logout {
-            margin-bottom: 50px;
-        }
-    }
 }
```

Comment:
I believe this got accidentally deleted.

Reply:
yes, sorry about that. it's clean now